src/Skcore.jl

```diff
@@ -115,7 +114,42 @@ symbols_in(e::Expr) = union(symbols_in(e.head), map(symbols_in, e.args)...)
 symbols_in(e::Symbol) = Set([e])
 symbols_in(::Any) = Set()
 
+import_already_warned = false
+function import_sklearn()
+    global import_already_warned
+
+    @static if Sys.isapple()
+      try
+        mod = pyimport("sklearn")
+      catch
+          @info "Installing non-mkl versions of sci-kit learn"
+
+          try
+            if PyCall.conda
+            #use non-mkl versions of python packages when Conda is used
+            #when a different non-conda local python is used everthing works fine
+              Conda.add("nomkl")
+              Conda.rm("mkl")
+            end
+            #PyCall installs scikit-learn using it's internal logic
+            mod = PyCall.pyimport_conda("sklearn", "scikit-learn")
+          catch
+              error("scikit-learn isn't properly installed."*
+                    "Please use PyCall default Conda or non-conda local python")
```

Comment:
This will make it difficult to help users who post an issue with this generic error message. How about

```suggestion
              @info("scikit-learn isn't properly installed."*
                    "Please use PyCall default Conda or non-conda local python")
              rethrow()
```

That way we'll have a more informative stack trace to work with.